src/main/java/com/cgdp/library/validation/Validator.java

```diff
@@ -0,0 +1,40 @@
+package com.cgdp.library.validation;
+
+import static com.cgdp.library.validation.IsbnValidator.isValidIsbn;
+
+import com.cgdp.library.exceptions.ValidationException;
+import java.time.LocalDate;
+import java.util.function.Supplier;
+
+public class Validator {
+
+    public static Long required(String paramName, Long value) {
```

Comment:
Why is value of type Long? We can simply use `Object` and this method will work for everything